sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/continuous/ContinuousMemoryStream.scala

```diff
@@ -0,0 +1,212 @@
+/*
```

Comment:
I think the consistent naming makes sense for the sources where they're automatically injected in planning, since they both have the same .format() option and thus the same name in some sense.

For memory streams it's different; the user explicitly instantiates the reader. So I think that making the names look similar is confusing, and I don't think that "MemoryStreamMicroBatchReader" is a good name for something explicitly instantiated.

Reply:
However, I stand by the other point of all the sources being colocated in streaming/sources. You missed changing that.